recipes/ggd/post-link.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -eo pipefail -o nounset
+
+## Organize required conda channels
+conda config --add channels defaults
+conda config --add channels ggd-genomics
+conda config --add channels bioconda
+conda config --add channels conda-forge
+
+## initialize local metadata
+cat << EOF > initialize_metadata.py
+"""
+Set up initial ggd local metdata
```

Comment:
*IF* you write files into or generally alter the environment via post-link, please also add an accompanying pre-unlink script to revert those actions.

Then again, as noted above:
General advice: If anyhow possible, avoid post-link scripts.

Reply:
Ideally, we'd have a pre-unlink script here. If not, a `conda remove ggd` won't remove files created here since `conda` does not keep track of post-link created files.

I'd like to encourage you to consider/look into how to revert the post-link actions in a pre-unlink script. However, if this is not easily doable right now, I won't block your PR on this.